common/src/main/java/org/dousi/utils/StringUtils.java

```diff
@@ -0,0 +1,17 @@
+package org.dousi.utils;
+
+import java.util.Random;
+
+public class StringUtils {
+
+  public static String getRandomString(int length) {
```

Comment:
I think it's better to have a class called `SessionId` instead of string:
```java
class SessionId {
    private byte[] id;
    
    public byte[] get bytes() {
    }

    public static fromRandom() {
         // random
          id = xxx;
    }
}
```